cvat/apps/annotation/README.md

```diff
@@ -150,3 +150,352 @@ It allows to download and upload annotations in different formats and easily add
+  Short description of  `*.txt` file structure: each line describes label and bounding box
+  in the following format `label_id cx cy w h`.
+  `obj.names` contains the ordered list of label names.
+- supported shapes - Rectangles
+
+#### Yolo loader description
+-   uploaded file: a zip archive with following structure:
+    ```bash
+    taskname.zip
+    ├── frame_000001.txt
+    ├── frame_000002.txt
+    ├── frame_000003.txt
+    ├── ...
+    └──obj.names
+    ```
+    It should be possible to match the CVAT frame(imagename) and annotation filename
+    There are 2 options:
+    1. full match between image name and name of annotation `*.txt` file
+       (in case of a task was created from images or archive of images).
+    1. match by frame number (if CVAT cannot match by name). File name should be in the following format `frame_%6d.jpg`.
+       It will be used when task was created from a video.
+
+-   supported shapes: Rectangles
+-   additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+#### How to create a task from YOLO formatted dataset (from VOC for example)
+1. Follow the official [guide](https://pjreddie.com/darknet/yolo/)(see Training YOLO on VOC section)
+   and prepare the YOLO formatted annotation.
+1. Zip train images
+   ```bash
+   zip images.zip -j -@ < train.txt
+   ```
+1. Create a CVAT task with the following labels:
+   ```bash
+   aeroplane bicycle bird boat bottle bus car cat chair cow diningtable dog horse motorbike person pottedplant sheep sofa train tvmonitor
+   ```
+   Select images.zip as data. Most likely you should use `share`
+   functionality because size of images.zip is more than 500Mb.
+   See [Creating an annotation task](cvat/apps/documentation/user_guide.md#creating-an-annotation-task)
+   guide for details.
+1. Create `obj.names` with the following content:
+   ```bash
+   aeroplane
+   bicycle
+   bird
+   boat
+   bottle
+   bus
+   car
+   cat
+   chair
+   cow
+   diningtable
+   dog
+   horse
+   motorbike
+   person
+   pottedplant
+   sheep
+   sofa
+   train
+   tvmonitor
+   ```
+1. Zip all label files together (we need to add only label files that correspond to the train subset)
+   ```bash
+   cat train.txt | while read p; do echo ${p%/*/*}/labels/${${p##*/}%%.*}.txt; done | zip labels.zip -j -@ obj.names
+   ```
+1. Click `Upload annotation` button, choose `YOLO ZIP 1.0` and select the *.zip file with labels from previous step.
+   It may take some time.
+
+### [MS COCO Object Detection](http://cocodataset.org/#format-data)
+#### COCO dumper description
+- downloaded file: single unpacked `json`. Detailed description of the MS COCO format can be found [here](http://cocodataset.org/#format-data)
+- supported shapes - Polygons, Rectangles (interpreted as polygons)
+
+#### COCO loader description
+- uploaded file: single unpacked `*.json`.
+- supported shapes: Polygons (the `segmentation` must not be empty)
```

Comment:
Is it true now?

Reply:
At the moment yes, I guess this description should be changed in the corresponding PR(https://github.com/opencv/cvat/pull/788)